server/utils/getNODE_ENV.js

```diff
@@ -0,0 +1,15 @@
+/* eslint-disable no-console */
+const getNODE_ENV = () => {
+  try {
+    if (process.env.NODE_ENV === undefined)
```

Comment:
if (!process.env.NODE_ENV) should be enough

Reply:
Switched to that in `jwt-authentication-login-page`.